### ISSUE TITLE
The examples for aria-expanded differ in the Wiki....

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#126]
From: @overtune 

Not shure if this is the place to ask, but I found (maybe) a inconsistency in the Wiki.
On this page the ”aria-expanded” attribute is set on the section that expands/collapse:
http://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region

But on this page the ”aria-expanded” attribute is set on the button:
http://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element

Perhaps both ways works?

**Comment by overtune** [comment]
Great! I'm the one that sent the original email (Johan).
Perhaps you could remove my email-adress from the issue, so I don't get to much spam from bots...
I will just follow the progress here.
Thanks!

**Comment by mraccess77** [comment]
I agree this is a mistake.  The aria-expanded should be on the button.  Since this is on the wiki on a page that is not meant to be authoritative I'm happy to update unless people don't agree the aria-expanded should be on the button.

**Comment by awkawk** [comment]
@mraccess77 that would be fine with me.  I'm not sure how this technique got stalled, it looks like it was accepted by the TF in 2012 but I don't recall it in the full WG discussion.

It looks like the code came directly from the OpenAjax site and that is where the live example is at.  Perhaps @jongund might be able to get the OpenAjax page corrected?

**Comment by alastc** [comment]
Closing, we don't maintain (or encourage the use of) techniques on the wiki any more, and I can't find a live equivelent in the [techniques list](https://www.w3.org/WAI/WCAG21/Techniques/).
